### ISSUE TITLE
Glutamate dehydrogenase antigen present

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#11086]
**Preferred term label:**
Glutamate dehydrogenase antigen present

**Synonyms**
GDH 

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Clostridioides difficile infection

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6001278
- label: Positive stool glutamate dehydrogenase test
- parents: [Positive stool infectious agent test (HP:0034866)](https://hpo.jax.org/app/browse/term/HP:0034866)
- pmid: PMID:24145018|PMID:35634999
- def: Detection of the presence of glutamate dehydrogenase produced by Clostridium difficile in a stool sample.
- comment: Clostridium difficile produces an NAD-specific glutamate dehydrogenase (GDH), which converts l-glutamate into alpha-ketoglutarate through an irreversible reaction. The enzyme GDH is detected in the stool samples of patients with C. difficile-associated disease and serves as one of the diagnostic tools to detect C. difficile infection. Clostridium difficile is the leading cause of hospital-acquired diarrhoea, which ranges in severity from mild diarrhoea to fulminant colitis.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/11086
- synonyms: Detection of C. difficile glutamate dehydrogenase(NONE)|Positive C. difficile glutamate dehydrogenase test(NONE)

**Comment by MickeySegal** [comment]
The name Clostridium has been replaced by Clostridioides, so you should at least include the new name.
